src/assets/schemas/config.schema.json

```diff
@@ -505,6 +501,9 @@
         },
         "placeholders": {
           "$ref": "#/definitions/placeholderLabels"
+        },
+        "tableColumnHeaders": {
```

Comment:
What's the tableColumnHeaders? where does it renders? Is it the title of the actions modal?

Reply:
Yes. I made it that if by any chance one day another generic column gets added.